email.d

```diff
@@ -1172,6 +1167,20 @@ string encodeBase64Mime(const(ubyte[]) content, string LINESEP = "\r\n") {
 	return cast(immutable(char[]))content.chunks(SOURCE_CHUNK_LENGTH).base64encode.join(LINESEP);
 }
 
+/// Base64 range decoder UFCS helper.
+alias base64decode = Base64.decoder;
+
+/// Base64 decoder, ignoring linebreaks which are mandated by RFC2045
+immutable(ubyte[]) decodeBase64Mime(string encodedPart) {
+	return cast(immutable(ubyte[])) encodedPart
```

Comment:
i hate these range pipelines so much

since im already using enough phobos in here it doesn't really harm anything though so might as well keep it

Reply:
I'll try to remember for next time